docs/home/howto/PATCHING.md

```diff
@@ -0,0 +1,79 @@
+# Patching OS with running Epiphany components
+This guide describes steps you have to perform to patch RedHat systems in a way to don't interrupt working Epiphany components.
+
+### Disclamer
+We providing a recommended way to patch your RedHat systems. Before proceeding with patching the production environment we highly recommending performing an update of your test cluster first.  
+This document will help you decide how you should patch your OS. This is not a step-by-step guide.
+
+### Requirements:
+- The fresh, actual backup contains your all sensitive data
+
+## AWS
+### Suggested OS images
+For current Epiphany version we recommend the following image (AMI):  
```

Comment:
Make it explicit which you mean, in case we missed updating this document during releases.